app/controllers/events_controller.rb

```diff
@@ -8,6 +8,11 @@ def index
   end
 
   def show
+    case event.status
```

Comment:
This code will not give you the right translations. In my opinion you don't have access to `event` here. You have only across to `@event`. But still this is not the place where we should have translations.